tests/data/expected_jsonschemas/expected.MockComponent.schema.json

```diff
@@ -1,264 +1,258 @@
+        {
+          "type": "array",
+          "title": "array"
+        }
+      ],
+      "description": "TBD"
     },
-    "Check Int": {
+    "Check Regex Integer": {
       "description": "TBD",
-      "minLength": 1,
       "not": {
         "type": "null"
       }
     },
-    "Check List": {
+    "Check Unique": {
       "description": "TBD",
-      "minLength": 1,
       "not": {
         "type": "null"
       }
     },
-    "Check List Enum": {
-      "description": "TBD",
-      "items": {
-        "enum": [
-          "ab",
-          "ef",
-          "cd",
-          "gh"
-        ]
-      },
-      "maxItems": 4,
-      "type": "array"
-    },
-    "Check List Enum Strict": {
-      "description": "TBD",
-      "items": {
-        "enum": [
-          "ab",
-          "ef",
-          "cd",
-          "gh"
-        ]
-      },
-      "maxItems": 4,
-      "type": "array"
-    },
-    "Check List Like": {
+    "Check Date": {
       "description": "TBD",
-      "minLength": 1,
       "not": {
         "type": "null"
       }
     },
-    "Check List Like Enum": {
+    "Check Num": {
       "description": "TBD",
-      "items": {
-        "enum": [
-          "ab",
-          "ef",
-          "cd",
-          "gh"
-        ]
-      },
-      "maxItems": 4,
-      "type": "array"
+      "type": "number"
     },
-    "Check List Strict": {
+    "Check Float": {
       "description": "TBD",
-      "minLength": 1,
-      "not": {
-        "type": "null"
-      }
+      "type": "number"
     },
-    "Check Match Exactly": {
+    "Check Match at Least values": {
       "description": "TBD",
-      "minLength": 1,
       "not": {
         "type": "null"
       }
     },
-    "Check Match Exactly values": {
-      "description": "TBD",
-      "minLength": 1,
-      "not": {
-        "type": "null"
-      }
+    "Check Regex List Like": {
+      "oneOf": [
+        {
+          "type": "array",
+          "title": "array"
+        }
+      ],
+      "description": "TBD"
     },
-    "Check Match None": {
-      "description": "TBD",
-      "minLength": 1,
-      "not": {
-        "type": "null"
-      }
+    "Check List": {
+      "oneOf": [
+        {
+          "type": "array",
+          "title": "array"
+        }
+      ],
+      "description": "TBD"
     },
     "Check Match None values": {
       "description": "TBD",
-      "minLength": 1,
       "not": {
         "type": "null"
       }
     },
-    "Check Match at Least": {
+    "Component": {
       "description": "TBD",
-      "minLength": 1,
       "not": {
         "type": "null"
       }
     },
-    "Check Match at Least values": {
+    "Check Match Exactly": {
       "description": "TBD",
-      "minLength": 1,
       "not": {
         "type": "null"
       }
     },
-    "Check NA": {
+    "Check Regex Single": {
       "description": "TBD",
-      "minLength": 1,
       "not": {
         "type": "null"
       }
     },
-    "Check Num": {
+    "Check Range": {
```

Comment:
if seems like initially we have `Check Range` as required but now it gets changed to optional?

Reply:
It doesn't look optional to me? It has type: "number", it doesn't have

```
"oneOf" : [
  {"type": "number},
  {"type": "null"}
]
```
And it's in the "required" list at the bottom.

Reply:
Sorry my bad. I saw it got listed in "required" here: https://github.com/Sage-Bionetworks/schematic/pull/1604/files/d4b52f71d5a868756276068bf841a6cd900c217e#diff-32abeaa43c00f1f8d0ed3b9b30145d4d2c66bc6550f9bf836006c73a2487e2b7R225